### PR TITLE
Add clock and uuid package

### PR DESCRIPTION
**Pull request by leonardojoao**

### CODE REVIEW

**Review by otavio — CHANGES_REQUESTED** [review]
As this is a new pkg we could be a good citizen and add the docstrings?

We could also document the package. So we start doing it for all new code and eventually do it to old code.

**Comment by leonardojoao** [comment]
> As this is a new pkg we could be a good citizen and add the docstrings?
> 
> We could also document the package. So we start doing it for all new code and eventually do it to old code.

What would the idea of ​​documentation be, or would there be an example of which model to follow?

**Comment by otavio** [comment]
@leonardojoao take a look at https://golang.org/src/go/doc/example.go

File: pkg/clock/clock.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import (
+	"time"
+)
+
+// Clock is an interface to store values ​​of type time
+type Clock interface {
+	Now() time.Time
+}
+
+// Get value from environment variable
```
Comment (otavio) [inline pkg/clock/clock.go:12]: environment?

**Review by otavio — APPROVED** [review]

**Review by otavio — COMMENTED** [review]
Typo on commit log "phg: use uuid package"

File: pkg/clock/clock.go (hunk 1 above)
Comment (leonardojoao) [reply pkg/clock/clock.go]: No, this is the defaultBackend which is used to configure the defaultBackend.

Comment (otavio) [reply pkg/clock/clock.go]: Ok; good. So it is not related to environment. Good. Closing it.

**Comment by otavio** [comment]
@eduardoveiga @leonardojoao I sent a lint to avoid those packages to ever to be used.

Some errors:

pkg/models/device.go:4:2: `time` is in the blacklist: time access must be done using 'github.com/shellhub-io/shellhub/pkg/clock' (depguard)
        "time"
        ^
pkg/models/license.go:3:8: `time` is in the blacklist: time access must be done using 'github.com/shellhub-io/shellhub/pkg/clock' (depguard)
import "time"
       ^
pkg/models/privatekey.go:3:8: `time` is in the blacklist: time access must be done using 'github.com/shellhub-io/shellhub/pkg/clock' (depguard)
import "time"

**Pull request merged**